curl-and-grep/pipelinerun.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: tekton.dev/v1alpha1
```

Comment:
We tend to have those only in the `README.md` (especially as the CI will try to apply those, which we may want or not) 
cc @dlorenc

Reply:
got it, i will submit another PR is you find the idea as usable and implementation as viable.

Reply:
let's wait for @dlorenc. You'll be able to amend your commit anyway, no need for a new PR :angel:

Reply:
Yep, sorry for missing this. +1 on moving to the readme :)

Reply:
thanks for the comment, I need some time to refresh my mind to make changes :)